### PR TITLE
Minor bug fixes

### PR DESCRIPTION
**Pull request by gowthamrao**
Add userCredentials.csv file to gitignore
Fix bug to sql annotation_id calculation  https://github.com/OHDSI/CohortDiagnostics/issues/887
Fix warning in orphan concept module https://github.com/OHDSI/CohortDiagnostics/issues/886

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@ tests/testthat/testthat-problems.rds
 renv.lock
 *.sqlite
 MergedCohortDiagnosticsData.7z
+*.csv
```
Comment (azimov) [inline .gitignore:27]: We use csv files for settings and ddls. Please remove from the gitignore

Comment (gowthamrao) [reply .gitignore]: done

**Comment by gowthamrao** [comment]
https://github.com/OHDSI/CohortDiagnostics/issues/880 fixes this issue with hasData function update

**Comment by gowthamrao** [comment]
Fixes https://github.com/OHDSI/CohortDiagnostics/issues/883

**Comment by gowthamrao** [comment]
This PR fixes some minor issues found during testing of the develop branch in Cohort Diagnostics. There are other issues that need a little bit more detailed look into the reactive objects.

**Comment by gowthamrao** [comment]
hasData change seems to have fixed this https://github.com/OHDSI/CohortDiagnostics/issues/882

File: inst/shiny/DiagnosticsExplorer/R/OrphanConceptsModule.R [hunk 2]
```diff
@@ -97,7 +97,7 @@ orphanConceptsModule <- function(id,
         data <- data %>%
           dplyr::filter(is.na(.data$standardConcept) |
                           (
-                            !is.na(.data$standardConcept) && .data$standardConcept != "S"
+                            any(!is.na(.data$standardConcept), .data$standardConcept != "S")
```
Comment (azimov) [inline inst/shiny/DiagnosticsExplorer/R/OrphanConceptsModule.R:100]: I assume the change from a logical "AND" to any (which is OR) fixes the issue with it loading?

**Review by azimov — APPROVED** [review]

**Review by azimov — APPROVED** [review]

**Comment by gowthamrao** [comment]
fixes https://github.com/OHDSI/CohortDiagnostics/issues/884

**Review by azimov — APPROVED** [review]

**Pull request merged**